waspc/data/Generator/templates/sdk/test/vitest/setup.ts

```diff
@@ -0,0 +1,8 @@
+import { afterEach } from 'vitest'
```

Comment:
I have this setup file in both web-app and sdk so I could experiment with importing it from different locations e.g. relatively from `.wasp/out/web-app/...` and `wasp/...` and `.wasp/out/sdk./..`